ingestion/functions/parsing/canada/canada.py

```diff
@@ -0,0 +1,84 @@
+import csv
+import os
+import sys
+from datetime import date, datetime
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+# Input format: '60-69' -> {"start": 60, "end": 69} 
+def convert_age(raw_age):
+    segments = raw_age.split("-")
+    if len(segments) != 2:
+        return None
+    return { "start": int(segments[0]) , "end": int(segments[1]) }
+
+def convert_location(health_region, province, country):
+    terms = []
+    if health_region != "Not Reported":
+        terms.append(health_region)
+    if province != "Not Reported":
+        terms.append(province)
+    if country != "Not Reported":
+        terms.append(country)
+    return {"query":  ", ".join(terms)}
+
+# dd-MM-yyyy -> yyyy-MM-dd
+def convert_date(raw):
+    segments = raw.split("-")
+    if len(segments) != 3:
+        return None
+    return "-".join(reversed(segments))
+
+def parse_cases(raw_data_file, source_id, source_url):
+    """
+    Parses G.h-format case data from raw API data.
+    """
+
+    def parse_csv_line(line):
+        """
+        Parse one csv line within the csv file.
+        """
+
+        date_report = convert_date(line[7]) # date_report
+
+        return {
+            "caseReference": {
+                "sourceId": source_id,
+                "sourceEntryId": line[0], # case_id
+                "sourceUrl": source_url
+            },
```

Comment:
Seems like there are some source URLs provided in the data. Could we add those as `additionalSources`?

![image](https://user-images.githubusercontent.com/63060924/94213826-e3047300-fea5-11ea-9219-422c9d8b5115.png)

Reply:
Yeah! Found that this field should be the combination of "case_source" and "additional_info". Updated.